docs/Bug-Trackers-and-Feedback-Channels.md

```diff
@@ -156,6 +158,26 @@ Jira tickets can be assigned to a user when they are created. This can be achiev
 
   E.g - http​&#65279;://companyname.checkmarx.com?assignee=someUsersEmail@&#65279;xyz.com
 
+### <a name="issueSummaryFormat">Configuring the Jira Issue Summary</a>
+
+The sast-issue-summary-format and sast-issue-summary-branch-format properties can be used to configure the issue summary of the issues that CxFlow creates in Jira for vulnerabilities detected by CxSAST. The following substitutions are performed on the properties’ values to generate the issue summary:
```

Comment:
CxFlow already has a place where textual templates are used: see the ConfigurationOverrider class. It uses another template format, e.g. `${branch}`. Maybe it's better to use the same format for Jira as well?

Reply:
I based my implementation on the mechanism we use to generate filenames (see the SanitizingFilenameFormatter class) so it would seem that CxFlow already has two places where textual templates are used :-).

I have a slight preference for what I have currently implemented because I have already implemented it.

Reply:
Ok, no problem :)